.eslintrc

```diff
@@ -100,6 +100,7 @@
     "max-lines": 0,
     "max-nested-callbacks": [2, 4],
     "max-params": [2, 6],
+    "max-statements": [2, 50],
```

Comment:
We have had some issues with some nit-like rules like this in the past. We ended up disabling the complexity rules in other repos because we'd add one more `if ()` statement and then have to add an inline disable of the rule or have to tweak the code to have it pass the linter.